tests/integration/url_generator_only/test_url_generator_only.py

```diff
@@ -50,3 +50,62 @@ def test_expected_integration_sample():
             '''
             assert queries[0].query_string != ""
 
+    
+def expected_test_for_get():
+    path_to_metadata = Path(__file__).parent.joinpath("metadata-northwind-v2.xml")
+    metadata_file_contents = path_to_metadata.read_bytes()
+    restrictions = RestrictionsGroup(None)
+    get_builder = DirectBuilder(metadata_file_contents, restrictions,"GET")
+    get_entities = get_builder.build()
+    queryable_factory = SingleQueryable
+    queries_list = []
+    queries_list.clear()
+    for queryable in get_entities.all():
+        entityset_urls_count = len(queryable.entity_set.entity_type.proprties())
+        for _ in range(entityset_urls_count):
+            q = queryable_factory(queryable, logger, 1)
+            queries = q.generate()
+            queries_list.append(queries[0].query_string)
+    queries_list=set(queries_list)
+    choice = queries_list.pop()
+    if("filter" in choice or "expand" in choice or "startswith" in choice or "replace" in choice or "substring" in choice or "inlinecount" in choice):
+        return True
+    else:
+        return False
```

Comment:
Change this logic to proper asserts. This is not a unit test.

Please read this chapter (but code uses unittest module, we use pytest, so don't copy-paste, only learn the logic behind) + this part of pytest documentation. In other words, first link is about understanding WHY, second link about understanding HOW.

https://diveintopython3.net/unit-testing.html
https://docs.pytest.org/en/3.0.4/assert.html

Reply:
ok